src/bridge_lcd.cpp

```diff
@@ -364,26 +364,18 @@ void bridge_lcd::init()
 #endif
 }
 
-void bridge_lcd::stop()
+void bridge_lcd::reinit()
 {
-#ifdef LCD_SSD1306
```

Comment:
Removing the conditional compilation here means that the same will apply to all three types (two basically).  Have you tested with both